### DISCUSSION TITLE
Auth 42 ?

### DISCUSSION THREAD

**Discussion by ghost** [miguelgrinberg/python-socketio#842] [Q&A]
updated

**Comment by miguelgrinberg** [comment]
Try removing the `/socket.io` portion of the URL.

**Comment by ghost** [comment]


**Reply by miguelgrinberg** [reply]
I'm sorry but I don't understand what you are asking. Please look at the logs of your server to see if there are any errors there. Also see the troubleshooting section of the python-socketio documentation.
